test/bouture.test.js

```diff
@@ -267,3 +267,29 @@ describe(`Bouture.ul( ['a', 'b', 'c'].map(text => Bouture.li(text)))`, () => {
     expect(ul.children[0].textContent).to.equal('a')
   })
 })
+
+describe(`Bouture.button({onClick: e => { called = false }})`, () => {
+  it('should create a button and bind with an onclick event', () => {
+    let called = false
+    const click = new MouseEvent('click')
+    const button = Bouture.button({onClick: e => { called = true }}).getElement()
+    button.dispatchEvent(click)
+
+    expect(called).to.be.true
+  })
+})
+
+describe(`Bouture.button({onceClick: 'Yo'`, () => {
+  it('should create a button and bind a click event who can only used once', () => {
```

Comment:
Maybe another equivalent test could be written for `onClick` to make sure that the listener is called twice. What do you think?